keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -713,11 +843,11 @@ BucketAccess {
     // BucketClaimName is the name of the BucketClaim.
     BucketClaimName string
```

Comment:
Hmm. This is a good idea. Before we started on v1alpha2, I was trying to see how we could accomplish this without API changes. But if we can modify the API, maybe it would be best. I'll think about this also, and probably include this suggestion. Thanks!

Reply:
We can also use validation on the API side, to ensure that the array has at least one item.